pkg/web/templates/partials/counterparty/counterparty_list.html

```diff
@@ -1,36 +1,36 @@
 {{- $canEditAccounts := not .IsViewOnly -}}
 {{- with .CounterpartyList -}}
 <div class="table-container pb-36 overflow-x-auto overflow-y-hidden">
-  <table class="table table-sm">
+  <table class="table">
     <thead class="font-bold text-base text-black">
       <tr>
         <th>Name</th>
         <th>Website</th>
         <th>Protocol</th>
         <td>Travel Address</td>
         <th>Date Added</th>
-        <th>Action</th>
+        <th></th>
       </tr>
     </thead>
     <tbody>
       {{ if .Counterparties }}
       {{ range .Counterparties }}
-      <tr class="hover">
+      <tr>
         <td>{{ .Name }}</td>
         <td>{{ .Website }}</td>
         <td>{{ .Protocol }}</td>
         <td class="max-w-sm break-words">{{ .TravelAddress }}</td>
         {{ $created := .Created.Format "2006-01-02T15:04:05-0700" }}
         <td class="datetime">{{ $created }}</td>
         <td>
-          <div class="dropdown">
-            <div tabindex="0" role="button" class="btn btn-ghost font-bold text-xl">&hellip;</div>
-            <ul tabindex="0" class="dropdown-content z-[1] menu p-2 shadow bg-black font-semibold text-white rounded-box">
-              {{ if and (eq .Source "user") ($canEditAccounts) }}
-              <li><a onclick="cparty_modal.showModal()" hx-get="/v1/counterparties/{{ .ID }}/edit" hx-target="#cparty_modal" hx-swap="innerHTML">Edit</a></li>
-              {{ end }}
-              <li><a>View</a></li>
-            </ul>
+          <div class="flex gap-x-2 items-center">
+            {{ if and (eq .Source "user") ($canEditAccounts) }}
+            <button type="button" class="btn btn-sm bg-primary text-white hover:bg-primary/90" onclick="cparty_modal.showModal()" 
```

Comment:
I'd like to note that I added the `hx-indicator` attribute here to display a loader component, but it didn't work. I'll create a follow on story to figure that out.